osx_environment.sh

```diff
@@ -29,18 +29,15 @@ if [ ! $BARRIER_BUILD_ENV ]; then
 
     elif command -v brew; then
         printf "Detected Homebrew\n"
-        QT_PATH=$(brew --prefix qt)
-        OPENSSL_PATH=$(brew --prefix openssl)
+        QT_PATH=$(brew --prefix qt5)
 
-        check_dir_exists "$QT_PATH" 'qt'
-        check_dir_exists "$OPENSSL_PATH" 'openssl'
+        check_dir_exists "$QT_PATH" 'qt5'
 
         export BARRIER_BUILD_BREW=1
-        export CMAKE_PREFIX_PATH="$QT_PATH:$CMAKE_PREFIX_PATH"
-        export LD_LIBRARY_PATH="$OPENSSL_PATH/lib:$LD_LIBRARY_PATH"
-        export CPATH="$OPENSSL_PATH/include:$CPATH"
-        export PKG_CONFIG_PATH="$OPENSSL_PATH/lib/pkgconfig:$PKG_CONFIG_PATH"
-
+        export CMAKE_PREFIX_PATH="/opt/procursus:$QT_PATH:$CMAKE_PREFIX_PATH"
+        export LD_LIBRARY_PATH="/opt/procursus/lib:$LD_LIBRARY_PATH"
+        export CPATH="/opt/procursus/include:$CPATH"
+        export PKG_CONFIG_PATH="/opt/procursus/lib/pkgconfig:$PKG_CONFIG_PATH"
```

Comment:
I know little about Procursus, but it seems strange to include Pocursus paths when Homebrew is detected.